src/orquestra/sdk/_base/_api.py

```diff
@@ -607,7 +607,7 @@ def get_artifacts(
 
     def get_logs(
         self,
-        tasks: t.Union[TaskRunId, t.List[TaskRunId]],
+        tasks: t.Union[TaskInvocationId, t.List[TaskInvocationId]],
```

Comment:
**non-blocking, thought**: Hot take: `tasks` and `only_available` arguments in the `sdk.WorkflowRun` methods have very little value now that we have `sdk.WorkflowRun.get_tasks()`. I'd simply get rid of them.

Reply:
FYI: I've done just that for `.get_artifacts()` in https://github.com/zapatacomputing/orquestra-workflow-sdk/pull/23